### ISSUE TITLE
better LineModel estimation 

### ISSUE THREAD

**Issue by Borda** [scikit-image/scikit-image#2427]
## Description
I had a look at the line model estimation an the representation by angle and shift does not look so good for me, I would rather use common Least Square Minimization.

## Way to reproduce
[If reporting a bug, please include the following important information:]
- [x] Code example
```
import numpy as np
import matplotlib.pylab as plt
from skimage.measure import LineModel

x = np.linspace(1, 2, 100)
y = 1.5 * x + 0.5
plt.plot(x, y, 'g', label='true')

y += (np.random.rand(len(y)) - 0.5) * 1.5
plt.plot(x, y, 'bx', label='noise')

lm = LineModel()
lm.estimate(np.array([x, y]).T)
y2 = lm.predict_y(x)
plt.plot(x, y2, 'r', label='LineModel')

x_space = np.array([x, np.ones(x.shape)]).T
# X = A\B is the solution to the equation AX = B computed
# by Gaussian elimination: inv(A) * B
est = np.linalg.pinv(x_space).dot(y)
y3 = est[0] * x + est[1]
plt.plot(x, y3, 'm', label='LeastSqaure')

plt.legend(loc=0)
plt.show()
```
- [x] Relevant images ![figure_1](https://cloud.githubusercontent.com/assets/6035284/21623526/16a9a514-d203-11e6-946a-5cc36912c8b1.png)

**Comment by jni** [comment]
Looking at your fit, I'm concerned that the previous method was buggy, rather than fundamentally flawed. Perhaps `scipy.odr` would be the best approach going forward? However, I'm not in any way opposed to merging #2433, I just wanted to get a discussion going.

**Comment by soupault** [comment]
Closed via #2433.
